arangod/Cluster/MaintenanceFeature.cpp

```diff
@@ -105,6 +109,39 @@ bool findNotDoneActions(std::shared_ptr<maintenance::Action> const& action) {
 
 }  // namespace
 
+arangodb::Result arangodb::maintenance::collectionCount(arangodb::LogicalCollection const& collection,
```

Comment:
This was moved here from `Cluster/SynchronizeShard.cpp` since it is now needed in two places.